crates/bevy_math/src/cubic_splines.rs

```diff
@@ -1332,11 +1382,34 @@ impl<P: VectorSpace> RationalSegment<P> {
     }
 }
 
-/// A collection of [`RationalSegment`]s chained into a single parametric curve.
+impl<P: VectorSpace> Curve<P> for RationalSegment<P> {
+    #[inline]
+    fn domain(&self) -> Interval {
+        // We don't presently enforce this invariant directly, but this is true.
+        // (Also, the knot span is not used by computations in `RationalSegment` internally.)
+        Interval::new(0.0, self.knot_span)
```

Comment:
Just marking this since it seems like you decided to change this to `Interval::UNIT` in the other comment

https://github.com/bevyengine/bevy/pull/15469/commits/8010ed0c88515e91f82a850bc3e4a78493ec39bb#r1778457614